CMakeLists.txt

```diff
@@ -99,7 +99,7 @@ foreach(item ${ARCH})
     endif()
 endforeach()
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
nitpick, can you also update the reference to C++11 in the README to C++14?

Reply:
Fixed.